.github/workflows/pa11y.yml

```diff
@@ -0,0 +1,32 @@
+name: pa11y
+ 
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
I think this will fail on pull requests that add new URLs, just have it run on schedule

Reply:
No probs, thanks. Removed these lines of code, will run on schedule only.